lib/push.js

```diff
@@ -78,7 +79,17 @@ var reasonToEvents = {
   }
 }
 
-module.exports = function (log, db) {
+module.exports = function (log, db, config) {
+  var vapid
+  if (config.vapidKeysFile) {
+    var vapidKeys = JSON.parse(fs.readFileSync(config.vapidKeysFile))
```

Comment:
I repeat my former comment that you can replace the double `JSON.parse(fs.readFileSync(` with a single `require(` here.